llvm/include/llvm/Support/RevngOptions.h

```diff
@@ -0,0 +1,13 @@
+#ifndef LLVM_SUPPORT_REVNGOPTIONS_H
+#define LLVM_SUPPORT_REVNGOPTIONS_H
+
+#include "llvm/Support/CommandLine.h"
+
+namespace llvm::revng {
+
+extern bool OverrideRevngDebugInformationPreservationStyle;
+extern cl::opt<bool, true> RevngDebugInformationPreservationStyle;
```

Comment:
I'd just call this `AggressivelyPreserveDebugInfo`

Reply:
Do you not want it labeled `revng` for some reason? I feel like that adds a lot of clarity.